src/utils/device.ts

```diff
@@ -65,11 +66,7 @@ const canAccessDeviceQuery = balenaApi.prepare<{ id: number }>({
 	id: { '@': 'id' },
 	url: `device(@id)/canAccess`,
 });
-export const canAccessDevice = (
-	device: DeviceInfo,
-	port: number,
-	auth?: Buffer,
-) =>
+const $canAccessDevice = (device: DeviceInfo, port: number, auth?: Buffer) =>
```

Comment:
I think you should add a custom cache resolve to use just `device.id` from `device` rather than the `is_connected_to_vpn` status as well